pkgs/games/dwarf-fortress/dfhack/default.nix

```diff
@@ -3,14 +3,62 @@
 , enableStoneSense ? false,  allegro5, libGLU_combined
 , enableTWBT ? true, twbt
 , SDL
+, dfVersion
 }:
 
+with lib;
+
 let
-  dfVersion = "0.44.12";
-  version = "${dfVersion}-r1";
+  dfhack-releases = {
+    "0.43.05" = {
```

Comment:
Based on what criteria are releases added/removed here?

Reply:
I would prefer to have less versions then listed here, because it makes the build matrix one has to test rather big.

Reply:
@matthewbauer added them in #43085. I agree, some of these versions are rather old.